etc/config/swift.amazon.properties

```diff
@@ -42,6 +42,9 @@ compiler.swift55.exe=/opt/compiler-explorer/swift-5.5/usr/bin/swiftc
 compiler.swift55.semver=5.5
 compiler.swift56.exe=/opt/compiler-explorer/swift-5.6/usr/bin/swiftc
 compiler.swift56.semver=5.6
+compiler.swift57nightly.exe=/opt/compiler-explorer/swift-5.7/usr/bin/swiftc
+compiler.swift57nightly.semver=5.7
+compiler.swift57nightly.name=5.7 nightly
```

Comment:
It seems the `name` is ignored when `isSemVer=true`; the name is then built from the `baseName` and the `semver` - sorry for the confusion. I'll make a change to fix this